src/i2c.rs

```diff
@@ -551,7 +551,6 @@ pub enum Error {
     OVERRUN,
     NACK,
     TIMEOUT,
-    BUS,
```

Comment:
Can we leave this in there? I'd rather not break existing applications for no good reason.

Reply:
Yeah, I'm happy to. I'll add a ntoe that this error type is only maintained for backwards compatibility.